test/db/cmd/feat_redirect

```diff
@@ -2,9 +2,9 @@ NAME="pb>x;!wc -c x"
 FILE=malloc://1024
 CMDS=<<EOF
 p8 10 > dump
-!wc -c dump|awk "{print \$1}"
+!!wc -c dump|awk "{print \$1}"
```

Comment:
Does this need to be updated too then?

Reply:
Yeah all these changes can be reverted back if we are ok with the auto detection.